src/app/components/blog/components/blog-page/blog-page.component.ts

```diff
@@ -55,6 +55,30 @@ export class BlogPageComponent implements OnInit {
           });
           break;
       }
```

Comment:
These are place holders for when the other branch is merged

Reply:
Sounds good! It is merged now, but you can make the change in a future PR